modin/pandas/base.py

```diff
@@ -2824,19 +2810,18 @@ def values(self):
         return self.to_numpy()
 
     def __getattribute__(self, item):
+        attr = super().__getattribute__(item)
```

Comment:
Why does this need to move/change?

Reply:
I doesn't _exactly_ need to change (though it could help with inheriting `BasePandasDataset` from something else, not `object`, which has `__getattribute__()` defined - a proper inheritance is you defer to your ancestor).

As for moving - this is a small optimization, as we're getting the attribute twice in an ordinary case - once to check if it's a method, and for a second time to return the result. There's no need to get it twice, it shouldn't change between the calls :)

Reply:
That makes sense.